### DISCUSSION TITLE
How to move a pixel art without it getting blurred?

### DISCUSSION THREAD

**Discussion by Firefly65536** [keiwando/toonsquid#440] [Q&A] [answered]
Since pixel art contains very few pixels, when I move the art it gets blurred (translation, no rotation or scaling). Any tips on how to fix this?

**Comment by Firefly65536** [comment, marked as answer]
I figured it out, apparently I just have to choose Nearest instead of Bilinear in the Resizing Filter

**Reply by wandrinsheep** [reply]
I’m still having issues with blurring no matter what
